tests/context.cpp

```diff
@@ -12,3 +17,13 @@ TEST_CASE("context create, close and destroy", "[context]")
     context.close();
     CHECK(NULL == (void *) context);
 }
+
+#ifdef ZMQ_CPP11
+TEST_CASE("context swap", "[context]")
+{
+    zmq::context_t context1;
+    zmq::context_t context2;
+    swap(context1, context2); // found by adl
+    zmq::swap(context1, context2);
```

Comment:
Should this really be tested? Isn't it sufficient, if it works like this:
```
using std::swap;
swap(context1, context2);
```
This will work either by using a default overload from std or a custom overload using ADL from zmq.